infra/main.tf

```diff
@@ -47,9 +47,10 @@ data "google_client_config" "default" {
 }
 
 locals {
-  resource_path = "resource"
+  resource_path     = "resource"
+  firestore_db_name = "large-data-sharing-${random_id.random_code.hex}"
```

Comment:
udpate this to solution name for deployment

Reply:
Ok, I updated all the lds references aside from the ones in the go api.